src/re_frame/subs.cljc

```diff
@@ -183,14 +183,18 @@
                              f)
 
                          ;; one sugar pair
-                         2 (fn inp-fn
-                             ([_] (subscribe (second input-args)))
-                             ([_ _] (subscribe (second input-args))))
+                         2 (let [[label vec] input-args]
```

Comment:
I'm not that keen on `label` here, but I'm struggling to think of a different name. Perhaps `directive`?